matlab/schemas/+anatomy/Area.m

```diff
@@ -0,0 +1,13 @@
+%{
+# LUT for Brain Areas
+brain_area      :  enum('V1','P','POR','PM','AM','A','RL','AL','LI','LM')      # area name
+---
+%}
+
+classdef Area < dj.Lookup
```

Comment:
there is also a similar table called experiment.BrainArea, are both necessary?

Reply:
Its better if we move them to anatomy schema.